source/controller.cpp

```diff
@@ -125,4 +132,28 @@ bool GZ_getButtonHold(int idx, int phase) {
     } else {
         return false;
     }
+}
+
+void GZ_readZombieHoverInputs() {
+    u32 zh_window_length = 30;
+    u16 current_input = GZ_getButtonStatus();
+
+    if (cCt_getFrameCount() % zh_window_length == 0) {

```

Comment:
I'm not sure if it's a good idea to check if a second has passed this way because if for whatever reason this function does not get called on the 30th frame of a window, then the counter won't reset and it will spike up to above 15.  Doing something like "if (currentFrame - startOfWindowFrame) >= 30" would resolve this problem, but would require an extra variable to keep track of the start of the current window.